pages/common/babeld.md

```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing daemon for Babel which uses firewall-style filters.
+> More information: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Start babeld with a specific configuration file:
+
+`babeld -c {{path/to/babeld.conf}}`
+
+- Start babeld with multiple configuration file (read in order):
+
+`babeld -c {{path/to/ports.conf}} -c {{path/to/filters.conf}} -c {{path/to/interfaces.conf}}`
+
+- Start babeld and daemonise afterwards:
+
+`babeld -D`
+
+- Start babeld and pass a configuration command:
+
+`babeld -C {{'redistribute metric 256'}}`
+
+- Start babeld and specify on which interfaces to operate:
```

Comment:
```suggestion
- Start babeld and specify which network interfaces to operate:
```

Reply:
Maybe `Start babeld and specify which network interfaces to operate on:`. I think with the example and the fact that it is a routing daemon, it is understandable that babeld are dealing with network interfaces. I made the sentence so short that it looks nicer.

Reply:
Yes, you are right, edited to include the on.